specification/BUILDING.md

```diff
@@ -0,0 +1,110 @@
+<!-- omit in toc -->
+# Build instructions
+
+- [Building the specification](#building-the-specification)
+  - [Asciidoctor setup](#asciidoctor-setup)
+  - [Generating HTML and PDF with AsciiDoc](#generating-html-and-pdf-with-asciidoc)
+  - [Compressing the PDF](#compressing-the-pdf)
+- [A note about section IDs](#a-note-about-section-ids)
+- [Cross-linking between files](#cross-linking-between-files)
+
+## Building the specification 
+
+The following is a short summary of the basic process for generating a single HTML- or PDF file containing the whole specification.
+
+> Note: This process might be extended in the future, with details about how [wetzel](https://github.com/CesiumGS/wetzel) is used to generate the property reference. Depending on the exact process, the required toolchain might be summarized in a Docker container, similar to [Vulkan](https://github.com/KhronosGroup/Vulkan-Docs/blob/15d807ce4839d8feb523ca5c133a42a2aa448ade/BUILD.adoc), and controlled via a Makefile.
+
+### Asciidoctor setup
+
+- Install the Ruby interpreter, 2.3 or later, from http://www.ruby-lang.org/
+- Install Asciidoctor: `gem install asciidoctor`
+- In order to be able to generate PDF output: `gem install asciidoctor-pdf`
+- Install some rogue software: `gem install rouge` - no worries, that's the syntax highlighter...
+
+- A VSCode plugin for AsciiDoc syntax highlighting and preview: https://marketplace.visualstudio.com/items?itemName=asciidoctor.asciidoctor-vscode
+
+### Generating HTML and PDF with AsciiDoc
+
+- Generating HTML:
+  - `asciidoctor --verbose Specification.adoc --out-file Specification-1.1.0.html`
+- Generating PDF:
+  - `asciidoctor-pdf --verbose --trace --attribute scripts=cjk --attribute pdf-theme=default-with-fallback-font Specification.adoc --out-file Specification-1.1.0.pdf`
+  
+    > Note: The last call is derived from https://asciidoctor.org/docs/asciidoctor-pdf/#support-for-non-latin-languages and handles Japanese characters. Without them, the call could just be `asciidoctor -r asciidoctor-pdf -b pdf Specification.adoc -o Specification-1.1.0.pdf`
+
+<sup>If the call does not appear to do anything, neither generate a PDF nor print an error message, make sure you typed `asciidoctor` and not just `asciidoc`</sup>
+
+
+### Compressing the PDF
+
+By default, the PDF file is large, due to the large image files being inserted uncompressed. There are tools for compressing the PDF, including a dedicated `asciidoctor-pdf-optimize` tool, but they have caveats. 
```

Comment:
After compression some of the diagrams become difficult to read.


![Selection_375](https://user-images.githubusercontent.com/915398/170700691-7e60be1b-eebd-4139-a049-4db3ec488ae4.png)
![Selection_376](https://user-images.githubusercontent.com/915398/170700693-d65af9b7-1559-45a4-9014-3e2f673bb37b.png)

The PDF you posted in https://github.com/CesiumGS/3d-tiles/pull/682#issuecomment-1138670614 doesn't seem to be nearly as bad. That one is 4.4MB whereas the one I generated was 2.3MB. I'm not sure what would explain the difference.

Reply:
The build instructions mention the compression modes. The one in the example is `screen`, which is _very_ agressive in terms of reducing the image resolution. The one that I used for the PDF was `ebook`, which may be more appropriate here.

(I did not yet try them out all, and I think they are only "shortcuts" for a bunch of other parameters, including _specific_ parameters that govern the DPI settings for images and the compression/quality tradeoff for images).

Reply:
I'll consider this as 'resolved' for now (using the `/ebook` settings, as described in the updated build instructions)